packages/wrangler/src/config.ts

```diff
@@ -36,8 +36,6 @@ export type Config = {
    * The entrypoint/path to the JavaScript file that will be executed.
    *
    * @optional
-   * @inherited
```

Comment:
Are we saying this is not inherited??

Reply:
I... think so? But I'm not sure. It probably shouldn't be be redefined in any of the environments, but whether it should be available in them... I dunno tbh. Happy if we take a call one way or the other.

Reply:
Ok, imo think it should be inherited, but not be redefined.